maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -34,14 +19,40 @@
 import java.sql.SQLException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
+
+import net.lingala.zip4j.core.ZipFile;
```

Comment:
Is there any reason for not using java.util.zip.ZipFile?

Reply:
It more convenient to do this with zip4j.